src/libdrakvuf/libdrakvuf.h

```diff
@@ -170,7 +170,7 @@ typedef enum memaccess_type
 
 typedef struct process_data
 {
-    const char* name;   /* Process name */
+    char* name;         /* Process name */
```

Comment:
This must remain `const` here.

Reply:
Why are you marking this resolved when the field is still not `const`?

Reply:
What I mean for "introduce a non-const" version for the process name is to have it local to the event callbacks in `libdrakvuf/vmi.c`, ie a variable on the stack there. This field should never be exposed to the plugin side as non-const.

Reply:
@tklengyel I'm very sorry but I have forgotten to stage file (at the end of the working day :) )

Reply:
OK, no problem :)